README.md

```diff
@@ -78,6 +78,7 @@ $ ./pgremapper balance-bucket <bucket> [--max-backfills <n>] [--target-spread <n
 ```
 
 * `<bucket>`: A CRUSH bucket that directly contains OSDs.
+* `--device-class`: device class filter, balance only OSDs with this device class.
```

Comment:
For consistency, the sentence should be capitalised (i.e. s/`device`/`Device`/ at the start).